src/Components/Card/CustomCard.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Container } from '@material-ui/core';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import {maxWidth,display} from '@material-ui/core'
+
+const CustomCard = ({title,img,description}) => {
+
+    return (
+        <Container sx={{ display: 'inline-flex' }}>
+            <Card sx={{ maxWidth: 345 }}>
```

Comment:
the styles should come from Styles folder. Its not a blocking but is a note for next PR.

Reply:
Perfect, ill be aware of this on future PR.